cookbooks/cdo-varnish/templates/default/default.vcl.erb

```diff
@@ -497,9 +503,9 @@ end
     } else if (req.url ~ "^/auth") {
       set req.http.Cookie = regsuball(req.http.Cookie, ";(cdo-private|hour_of_code|NO_CACHE)=", "; \1=");
     } else if (req.http.host == "<%= canonical_hostname('learn.code.org') %>" || req.http.host == "<%= canonical_hostname('studio.code.org') %>") {
-      set req.http.Cookie = regsuball(req.http.Cookie, ";(language_|<%= session_cookie %>|<%= storage_cookie %>|hour_of_code|NO_CACHE)=", "; \1=");
+      set req.http.Cookie = regsuball(req.http.Cookie, ";(language_|<%= session_cookie %>|<%= deprecated_storage_cookie %>|<%= storage_id_cookie %>|hour_of_code|NO_CACHE)=", "; \1=");
```

Comment:
Per offline discussion, the above won't work because the ruby gets run before the vcl. Leaving this as-is because I'm scared of refactoring without any tests in place, there isn't a lot of churn on this code anyway, etc.